Chummer/Classes/ColorManager.cs

```diff
@@ -316,6 +383,27 @@ private static Color InverseGetDarkModeVersion(Color objColor)
             return FromHsla(fltHue, fltNewSaturationHsl, fltNewLightness, objColor.A);
         }
 
+        private static Color GetBrightenedVersion(Color objColor)
+        {
+            // Built-in functions are in HSV/HSB, so we need to convert to HSL to invert lightness.
+            float fltHue = objColor.GetHue() / 360.0f;
+            float fltBrightness = objColor.GetBrightness();
+            float fltLightness = objColor.GetSaturation();
+            fltLightness = Math.Min(fltLightness + 0.2f, 1);
+            return FromHsva(fltHue, fltBrightness, fltLightness, objColor.A);
+        }
+
+        private static Color GetDimmedVersion(Color objColor)
+        {
+            // Built-in functions are in HSV/HSB, so we need to convert to HSL to invert lightness.
+            float fltHue = objColor.GetHue() / 360.0f;
+            float fltBrightness = objColor.GetBrightness();
+            float fltLightness = objColor.GetSaturation();
+            fltLightness = Math.Max(0, fltLightness - 0.2f);
```

Comment:
You sure you want to subtract 20 from the Saturation of the color instead of multiplying it by some value that is less than 1?

Reply:
I'll change this to use a factor (0.85f). Result looks quite similar, but maybe I don't fully catch the corner cases you have in mind.